oclint-json-compilation-database

```diff
@@ -72,17 +73,15 @@ if args.includes:
 if args.excludes:
     for exclusion_filter in args.excludes:
         source_list = source_list_exclusion_filter(source_list, exclusion_filter)
-source_paths = '"' + '" "'.join(source_list) + '"'
-oclint_arguments = ''
+oclint_arguments = [OCLINT_BIN, '-p', args.build_path]
 if args.oclint_args:
-    oclint_arguments = ' ' + ' '.join(args.oclint_args)
-debug_argument = ''
+    oclint_arguments += args.oclint_args
 if args.debug:
-    debug_argument = ' -debug'
-oclint_invocation = '"' + OCLINT_BIN + '"' + debug_argument + oclint_arguments + ' ' + source_paths
+    oclint_arguments.append('-debug')
```

Comment:
`debug` mode depends on whether `oclint` is built with debugging flag turned on, and in our release build, `-debug` flag doesn't exit. I have heard feedback that in release mode, when people use `-debug` in `oclint-json-compilation-database`, it results in crashes. Any suggestions you have to fix this?